pages/index.vue

```diff
@@ -1,5 +1,48 @@
 <template>
-  <div>
-    <Atlas />
+  <div class="flex flex-col items-center h-full">
+    <!-- Adding the selectors -->
+    <div class="flex flex-col justify-start gap-1">
+      <div v-for="(map, i) in maps" :key="i">
+        <span v-if="maps.length>1" class="text-xl"> {{ getLetter(i) }} </span>
+        <MapSelector v-model="map.path" />
+        <button v-if="maps.length!=1" class="mx-4" @click="maps.splice(i, 1)">
+          <font-awesome-icon :icon="['fas', 'trash']" />
+        </button>
+        <button v-if="i===maps.length-1" class="mx-4" @click="maps.push(clone(map))">
+          <font-awesome-icon :icon="['fas', 'plus']" />
+        </button>
+      </div>
+    </div>
+    <!-- Adding the maps -->
+    <div class="flex flex-grow w-full">
+      <span
+        v-for="(map, id) in maps"
+        :key="id"
+        class="bg-center bg-no-repeat bg-contain flex-grow"
+        :style="{backgroundImage: `url(${map.path})`}"
+      >
+        <p v-if="maps.length>1" class="text-xl m-24"> {{ getLetter(id) }} </p>
```

Comment:
When resizing the window, the letter in `<p>` could appear on top of the figure, which is mainly due to the fact that our figure is a background image rather than an independent image. Need to either fix it through some styling (is there any specific styling forcing the content to surround the background image?) or display our image as an independent one (but the pain of image styling will occur...).

Reply:
Yes, well spotted! I couldn't get the `img` tags to properly scale down, so that's why I reverted to bg images again. I couldn't think if a better way to align the letters, but I'll have another go at it. 

If there's no easy solution, I was considering to leave out the letters altogether, I'm not sure how necessary they are. Would that be okay with you (as a compromise)?

Reply:
> Yes, well spotted! I couldn't get the `img` tags to properly scale down, so that's why I reverted to bg images again. I couldn't think if a better way to align the letters, but I'll have another go at it.

Yes, I understand the pain. We can just keep the img as bg image.

> If there's no easy solution, I was considering to leave out the letters altogether, I'm not sure how necessary they are. Would that be okay with you (as a compromise)?

Indeed, I think it is not necessary to put a letter there, as the corresponding between menu bar and the image is quite straight forward, especially given the fact that the user will add extra menu bar and image themselves. And the letter, in my opinion, "pollute" the beauty of our clean design. Less is more! So I agree, let's just drop the letters and merge the pull request.